dashboard/app/views/layouts/_thank_donors_interstitial.html.haml

```diff
@@ -0,0 +1,24 @@
+
+#thank-donors-modal.modal{style: 'display: none;'}
+  %button#close-thank-donors-dialog.close{'data-dismiss' => 'modal', 'aria-label' => 'Close'}
+    %span{'aria-hidden' => 'true'}= '&times;'.html_safe
+  .modal-dialog
+    .modal-content.no-modal-icon
+      %h2.donor-header= t('donor_interstitial.title') + '...'
```

Comment:
Manually appending ellipses here may be a bad idea from a translations / RTL point of view. @Hamms what do you recommend?

Reply:
Sounds like I should just include the ellipsis in the translation string. Will update.